### DISCUSSION TITLE
How to set the transmit power used in Sionna RT for this tutorial?

### DISCUSSION THREAD

**Discussion by lufeiaixuexi** [NVlabs/sionna#557] [Q&A]
Hello, how to set the transmit power used in Sionna RT for this tutorial in the code?We want to set up a new transmitter and set its power. How do we implement it in the code, in dB or W？

**Comment by faycalaa** [comment]
Hi @lufeiaixuexi,

It looks like your question is similar to [this one](https://github.com/NVlabs/sionna/discussions/356). Could you check it out?
